### PR TITLE
add functions around holidays that land on weekends. add related tests

### PR DESCRIPTION
**Pull request by mrbeal15**

### CODE REVIEW

**Review by nlindley — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -33,13 +37,47 @@ const isMemorialDay = date => {
 // July 4
 const isIndependenceDay = date => date.month() === 6 && date.date() === 4;
 
+const isMondayFollowingHoliday = date => {
+
+  if (isMonday(date)) {
+
+    const tomorrow = Moment(date).subtract(1, 'day');
```
Comment (xingwang) [inline src/index.js]: This should be named "yesterday".

File: src/index.js [hunk 2]
```diff
@@ -33,13 +37,47 @@ const isMemorialDay = date => {
...
+  }
+
+  return false;
+};
+
+const isFridayPrecedingHoliday = date => {
+
+  if (isFriday(date)) {
+
+    const yesterday = Moment(date).add(1, 'day');
```
Comment (xingwang) [inline src/index.js]: This should be "tomorrow".

File: src/index.js [hunk 3]
```diff
@@ -33,13 +37,47 @@ const isMemorialDay = date => {
...
+
+    return isNewYears(tomorrow)
+      || isChristmas(tomorrow)
+      || isIndependenceDay(tomorrow);
+  }
+
+  return false;
+};
+
+const isMonday = date => Moment(date).day() === 1;
```
Comment (xingwang) [inline src/index.js]: Should go above the function using these?

**Review by xingwang — APPROVED** [review]
🏁

**Pull request merged**